src/messages/MClientReply.h

```diff
@@ -323,6 +323,8 @@ struct openc_response_t {
 WRITE_CLASS_ENCODER(openc_response_t)
 
 class MClientReply final : public MMDSOp {
+private:
+  static constexpr int HEAD_VERSION = 2;
```

Comment:
I'm not understanding the reason for a version bump; this message was already doing the conversion properly?